README.md

```diff
@@ -1 +1,27 @@
-# animal-stories
\ No newline at end of file
+# animal-stories
+
+# detect if a word is an animal
+
+animal_list = ["hare", "rabbit", "tortoise", "goat", "hound", "duck", "ducking", "fox", "lion", "boar", "ant", "grasshopper", 
+                "cat", "monkey", "dog", "mouse", "crow", "eagle", "wolf", "peacock"]
+
+def isAnimal(value):
+    if value in animal_list:
+        value = True    
+    else:
+        value = False
+        
+isAnimal("hare")
```

Comment:
There is a bug in isAnimal. Have you tried print(isAnimal("hare"))? In the console it prints out "None". But I expect it to print out "True".